alfasim_sdk_api/detail/api_pointers.h

```diff
@@ -239,4 +258,56 @@ typedef int (*get_flow_pattern_func)(
 get_flow_pattern_func get_flow_pattern;
 
 
+/**
+*   get_tracer_id
+*
+*   Retrieves the tracer ID given a tracer reference. A tracer reference may be obtained from the
+*   user input data (See get_plugin_input_data_reference API function for an example).
+*/
+typedef int (*get_tracer_id_func)(void* ctx, int* tracer_id, void* reference);
+get_tracer_id_func get_tracer_id;
+
+
+/**
+*   get_tracer_name_size
+*
+*   Retrieves the size of the tracer name, given a tracer reference. A tracer reference may be
+*   obtained from the user input data (See get_plugin_input_data_reference API function for an
+*   example).
+*/
+typedef int (*get_tracer_name_size_func)(void* ctx, int* tracer_name_size, void* reference);
+get_tracer_name_size_func get_tracer_name_size;
+
+
+/**
+*   get_tracer_name
+*
+*   Retrieves the tracer name, given a tracer reference. The tracer_name parameter must be a valid
+*   and pre-allocated memory region where the name string will be copied to. A tracer reference may
+*   be obtained from the user input data (See get_plugin_input_data_reference API function for an
+*   example).
+*
+*   Example usage:
+*   int tracer_name_size = -1;
+*   char tracer_name[100];
```

Comment:
bad practice, you should dynamically allocate the buffer using `tracer_name_size` 😉